channelserver/src/server.rs

```diff
@@ -394,18 +391,8 @@ impl Handler<Connect> for ChannelServer {
         // tell the client what their channel is.
         let jpath = json!({ "link": format!("/v1/ws/{}", chan_id),
                             "channelid": chan_id });
-        if msg
-            .addr
-            .do_send(TextMessage(MessageType::Text, jpath.to_string()))
-            .is_err()
-        {
-            warn!(
-                self.log.log,
-                "Could not send path to channel";
-                "channel" => chan_id,
-                "remote_ip" => remote
-            )
-        };
+        msg.addr
+            .do_send(TextMessage(MessageType::Text, jpath.to_string()));
```

Comment:
Do we want to keep this warn log message as there's now an `Addr::try_send` that returns an Error like the old `do_send` did.

Reply:
Ah, sounds good. Forgot that `try_send` was an option